OMtopogen/create_topog_refinedSampling.py

```diff
@@ -368,14 +368,15 @@ def main(argv):
         #Dardanells' constrict
         j1,i1=15616, 39166 #get_indices1D(topo_lons, topo_lats ,26.39, 40.14)
         topo_elvs[j1+1,i1]=topo_elvs[j1,i1]
-    #Read a target grid
+    #Read a target grid from mosaic file
+    print("\nReading target grid from: {}\n".format(gridfilename))
     targ_grid =  netCDF4.Dataset(gridfilename)
     targ_lon = np.array(targ_grid.variables['x'])
     targ_lat = np.array(targ_grid.variables['y'])
     #x and y have shape (nyp,nxp). Topog does not need the last col for global grids (period in x).
-    targ_lon = targ_lon[:,:-1]
-    targ_lat = targ_lat[:,:-1]
-    print(" Target mesh shape: ",targ_lon.shape)
+    targ_lon = targ_lon[:][::2,::2]
```

Comment:
@sanAkel  This is done later in the merge tool:
https://github.com/nikizadehgfdl/ocean_model_topog_generator/blob/master/OMtopogen/merge_topog_tiles.py#L198

The calculations in this tool were intended to be done on the finer supergrid. 

It is possible what you suggest could work, but if we start from the coarser T-cell grid as you suggest, the refining algorithm will probably do one more refinement anyway.

Thanks.